src/pages/proposals/create/StepPayout.vue

```diff
@@ -265,13 +265,31 @@ export default {
       return (this.cycleDurationSec / this.daoSettings.periodDurationSec).toFixed(2)
     },
     cashToken () {
-      return !this.toggle ? this.getFormatedTokenAmount(this.peg * (this.$store.state.dao.settings.settings_treasuryTokenMultiplier_i ? this.$store.state.dao.settings.settings_treasuryTokenMultiplier_i : 1), Number.MAX_VALUE) : this.getFormatedTokenAmount((this.peg * (this.$store.state.dao.settings.settings_treasuryTokenMultiplier_i ? this.$store.state.dao.settings.settings_treasuryTokenMultiplier_i : 1) / this.periodsOnCycle).toFixed(2), Number.MAX_VALUE)
+      return !this.toggle
```

Comment:
@arsenijesavic this piece of code is reused 3 times where difference is only in token multiplier ( eg `this.daoSettings.treasuryTokenMultiplier` ) 

Extract this code as `calculateToken(tokenMultiplier)` function and then just use it as `calculateToken(this.daoSettings.treasuryTokenMultiplier)`